packages/devextreme/testing/helpers/pointerMock.js

```diff
@@ -151,6 +151,13 @@
                 return this;
             },
 
+            _wheel: (config) => {
```

Comment:
I think our case is too specific for adding one more method to PointerMock. You can also see that there is no other "private" methods.
Let's just directly trigger necessary event